src/style.css

```
@@ -1 +1,50 @@
-/* Styles go here */
+.header {
+  margin: 0;
+  display: flex;
+  width: 100%;
+  flex-direction: row;
+  justify-content: space-around;
+  font-family: Roboto, Helvetica, sans-serif;
+  height: 60px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  /* width: 100%; */
+  display: flex;
+  justify-content: flex-end;
+}
+
+.img {
+  width: 40px;
+  height: 40px;
+  align-self: center;
+  cursor: pointer;
+}
+
+.ul {
+  margin: 0;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: stretch;
+}
+
+.li {
+  margin: 0 10px;
```

```suggestion
  margin-right: 20px;
```
Те, про що я писав коментом вище